src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Commands.Resources.Rest.Netcore.csproj

```diff
@@ -27,7 +27,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="2.0.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.9.1-preview" />
```

Comment:
I don't think we should downgrade the package in Resources.

Reply:
The version was not upgraded in our repo; only in their branch. I aligned the version that we upgraded in our repo (from 1.9.0-preview to 1.9.1-preview). I changed this from 1.9.0-preview to 2.0.0-preview in a previous PR to fix their branch. But, now, I've gotten it to work without upgrading everyone to 2.0.0-preview.